src/main/scala/cromiam/webservice/CromIamApiService.scala

```diff
@@ -7,10 +7,12 @@ import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
 import akka.http.scaladsl.marshalling.ToResponseMarshallable
 import akka.http.scaladsl.model.StatusCodes.{BadRequest, Forbidden, NotImplemented, Unauthorized}
 import akka.http.scaladsl.model.{HttpRequest, HttpResponse}
-import akka.http.scaladsl.server.Directives._
 import akka.http.scaladsl.server._
 import akka.http.scaladsl.unmarshalling.Unmarshal
 import akka.stream.ActorMaterializer
+import cats.instances.future._
+import cats.instances.list._
+import cats.syntax.traverse._
```

Comment:
The compiler flag is [already on](https://github.com/broadinstitute/CromIAM/blob/75f3170db3b8c0483ad1240c3648279791bbb75a/build.sbt#L22-L23).

I just ran `Optimize Imports ⌥⌘O` to canonically tidy up this file. These fancy-pants imports actually come from some scala-fu in https://github.com/broadinstitute/CromIAM/pull/13#discussion_r127582467